test/csuite/wt2695_checksum/main.c

```diff
@@ -321,6 +324,17 @@ main(int argc, char *argv[])
         }
     }
 
+    /*
+     * "Strobed" misalignments - test every combo of size/misalignment up to 16B.
+     */
+    for (length = 0; length < 16; length++) {
+        for (misalignment = 0; misalignment < 16; misalignment++) {
+            hw = __wt_checksum(&data_ff[misalignment], length);
+            sw = __wt_checksum_sw(&data_ff[misalignment], length);
+            check(hw, sw, length, "0xff: strobed");
+        }
+    }
+
```

Comment:
Afaict this does not check that the tail length is correctly calculated as the length is never > VX_MIN_LEN.

Reply:
Yep, this is aimed at the head being misaligned. I can add a tail check if you feel it's valuable.

Reply:
Discussed in person. Additional effort does not seem warranted.